dashboard/client/src/components/ActorTable.tsx

```diff
@@ -144,15 +145,16 @@ const ActorTable = ({
             {[
               "",
               "ID",
-              "Restart Times",
-              "Name",
               "Class",
-              "Function",
+              "Name",
+              "State",
+              "Uptime",
```

Comment:
nit: Is there a way to maybe add some front-end logic to inform the definition of uptime here? This means since the actor is created right?

Reply:
If needed, we could add a explanation icon next to the header
<img width="390" alt="Screen Shot 2022-11-17 at 10 37 54 AM" src="https://user-images.githubusercontent.com/9677264/202530361-b0769363-c14c-4946-9fa7-4ac745784d27.png">

Reply:
Let's handle it separately from this PR!